### PR TITLE
Consolidate smoke & stress tests and add some documentation

### PR DESCRIPTION
**Pull request by kenoir**
This change consolidates smoke & stress tests into a single directory and adds some more documentation.

### CODE REVIEW

File: smoke_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# smoke_tests
+
+This directory contains [artillery.io](https://artillery.io/) configuration to run post service deployment smoke tests.
+
+These tests should to run after a deployment to ensure that things are behaving as expected in the deployed environment.
```
Comment (alexwlchan) [inline smoke_tests/README.md:5]:
```suggestion
These tests should to run after a deployment to ensure that things are behaving as expected in the deployed environment.
```

"should to run"?

File: smoke_tests/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# Run smoke tests against stage
+yarn smokeTestCatalogueApiStage
+
+# Run smoke tests against prod
+yarn smokeTestCatalogueApiProd
+```
+
+## Running stress tests
+
+**Important:** Ensure that you are not going to put undue load on the production services (or dependent Elastic clusters)! Lack of due care could break the catalogue API!
```
Comment (alexwlchan) [inline smoke_tests/README.md:30]: How do I know I'm not going to put undue load on the prod services?

Possibly worth explaining how to interrupt a running test?

**Review by jamesgorrie — APPROVED** [review]

File: smoke_tests/README.md (hunk 1 above)
Comment (kenoir) [reply smoke_tests/README.md:5]: Going to fix this in an upcoming PR - fixing the docs elsewhere too.

File: smoke_tests/README.md (hunk 2 above)
Comment (kenoir) [reply smoke_tests/README.md:30]: As above, will address shortly.

**Pull request merged**